src/components/card-with-link/CardWithLink.tsx

```diff
@@ -7,7 +7,7 @@ import TitleWithDescription from '~/components/title-with-description/TitleWithD
 import { styles } from '~/components/card-with-link/CardWithLink.styles'
 
 interface CardWithLinkProps {
-  img: string
+  img: string | React.ReactNode
```

Comment:
`React.ReactNode` includes `string `
```suggestion
  img: string | React.ReactNode
```